Sprint-2/interpret/time-format.js

```diff
@@ -12,20 +12,25 @@ function formatTimeDisplay(seconds) {
     remainingSeconds
   )}`;
 }
-
+console.log(formatTimeDisplay(61));
 // You will need to play computer with this example - use the Python Visualiser https://pythontutor.com/visualize.html#mode=edit
 // to help you answer these questions
 
 // Questions
 
 // a) When formatTimeDisplay is called how many times will pad be called?
+// 3
 
 // Call formatTimeDisplay with an input of 61, now answer the following:
 
 // b) What is the value assigned to num when pad is called for the first time?
+// 0
 
 // c) What is the return value of pad is called for the first time?
+// 00
 
 // d) What is the value assigned to num when pad is called for the last time in this program?  Explain your answer
+// 1 - the last call to pad happens when formatting remainingSeconds
 
 // e) What is the return value assigned to num when pad is called for the last time in this program?  Explain your answer
+// 01
```

Comment:
What is the explanation here? Why is this the case? Talk through your answer methodically.